packages/api/src/derives/attestation/attestationRx.e2e.ts

```diff
@@ -0,0 +1,261 @@
+// Copyright 2019 Centrality Investments Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { filter, first } from 'rxjs/operators';
+
+import { ApiRx } from '@cennznet/api';
+import { AttestationValue, TypeRegistry } from '@cennznet/types';
+import testKeyring from '@polkadot/keyring/testing';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { cryptoWaitReady } from '@polkadot/util-crypto';
+
+import initApiRx from '../../../../../jest/initApiRx';
+import { H256 } from '@cennznet/types';
```

Comment:
should be grouped with the other `@cennznet/types` imports (shouldn't we have lint for this 🤔)

Reply:
https://eslint.org/docs/rules/ the recommended does not include - sort-imports... we can add it specifically as a rule in the config.

Reply:
cause it is included in test and we have skipped all **/*.e2e.ts

Reply:
made this change